examples/simple_connection.rb

```diff
@@ -0,0 +1,10 @@
+# tiny example so you can play with the sshkit or make a failing example for an issue
+require 'bundler/setup'
```

Comment:
Why is this needed?

Reply:
it's meant as a starting point to debug sshkit itself, so it needs to use the local checkout and not the installed gem